woosmapgeofencingcore/src/main/java/com/webgeoservices/woosmapgeofencingcore/WoosmapSettingsCore.java

```diff
@@ -326,6 +326,7 @@ public static String getDistanceUnits() {
     public static String WoosmapURL = "https://api.woosmap.com";
     public static String SearchAPIUrl = "%s/stores/search/?private_key=%s&lat=%s&lng=%s&stores_by_page=20";
     public static String DistanceAPIUrl = "%s/distance/distancematrix/json?mode=%s&units=%s&language=%s&origins=%s,%s&destinations=%s&private_key=%s&elements=duration_distance";
+    public static String DistanceAPIWithTrafficUrl = "%s/distance/distancematrix/json?mode=%s&units=%s&language=%s&origins=%s,%s&destinations=%s&private_key=%s&departure_time=now&elements=duration_distance";
```

Comment:
How are you managing method parameters?
https://developers.woosmap.com/api-reference/#operation/getDistanceMatrix